extensions/amp-audio/validator-amp-audio.protoascii

```diff
@@ -30,6 +30,10 @@ attr_lists: { # amp-audio attributes that are common to both AMP and A4A format.
   attrs: { name: "artist" }
   attrs: { name: "artwork" }
   attrs: { name: "controls" }
+  attrs: {
+    name: "controlsList"
+    value: "nodownload"
```

Comment:
there are more values than just `nodownload`, keep it as `attrs: { name: "controlList" }` to allow arbitrary values